conf/routes

```diff
@@ -90,5 +90,4 @@ GET            /setup/confirmedmailerror            @com.yetu.oauth2provider.con
 GET            /setup/confirmedmail/:mailToken      @com.yetu.oauth2provider.controllers.setup.SetupController.handleSignUp(mailToken)
 
 # Download setup app
-GET            /setup/download                      @com.yetu.oauth2provider.controllers.setup.SetupController.download
-
```

Comment:
Yeah.. maybe a line or something.. I can checkout routes file from master to avoid.. just a second..